constants/creator_groups.json

```diff
@@ -274,7 +274,28 @@
     "Yonsei University",
     "York University",
     "Zhejiang University",
-    "Zurich University of Applied Sciences"
+    "Universitas Gadjah Mada",
```

Comment:
Great! I know this is a pain, but could you:

(1) sort these alphabetically -- use python function to read the file, then re-write it with sorted values list for ease of repetition
(2) add these also to constants/creator_groups_by_country.json. Requires a little bit of research into where each of these organizations are based, but should be quick